libs/sdk-backend-tiger/src/backend/index.ts

```diff
@@ -314,6 +316,22 @@ export class TigerBackend implements IAnalyticalBackend {
 
         return this.authProvider.authenticate({ client: this.client, backend: this });
     };
+
+    private updateCapabilities() {
```

Comment:
The fact that this is async may lead to some unwanted surprises if you check them too soon :/  Is it really necessary to do the backend call here? I'm not sure if I understand the comment `This is just for rollout of export csv/xls` does it mean that we will remove it later?

Reply:
You're right, we are aware of this async consequence, but as this should be just temporary and removed later (see below), no changes which would generalize this were taken. As the defaults are "false", in the very rare case this could lead only to the delayed appearance of the export option.

Currently export is phased. CSV is deliered first, then xlsx would follow. We don't have feature flag for that as feature flags are not on tiger AIO. At the same time, it's (temporarily) controlled in tiger AIO by a env variable which tells if the export is allowed or not. So checking the api is the only way we could tell on AIO for which formats the service is ready.

After both csv and xlsx is delivered and after this env variable on Tiger is removed, we can safely remove this async check and have default capabilities to true as there will be no point in running aio without exports so ui and backend will be in sync.

Reply:
Added reference to the JIRA ticket